contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -41,7 +41,7 @@
 #include "../src/collation.h"
 #include "../src/rolecmds.h"
 
-#define TSQL_STAT_GET_ACTIVITY_COLS 24
+#define TSQL_STAT_GET_ACTIVITY_COLS 25
```

Comment:
We need to deprecate sysprocesses which is dependent on tsql_stat_get_activity. So if any cx object is dependent on sysprocesses then it would still point to deprecated version of sysprocesses after which is expecting tsql_stat_get_activity to return 24 columns and hence it would be failed. 

Alternate way to achieve this is to deprecate C function definition also which would be referenced by deprecated version of sysprocesses.